### PR TITLE
implement payment,order and changed line item and product's toString,…

### PR DESCRIPTION
**Pull request by tomerkel**
… changed Orders list in Account from hashmap to list

### CODE REVIEW

File: src/ImmediatePayment.java [hunk 1]
```diff
@@ -1,3 +1,13 @@
 public class ImmediatePayment extends Payment{
     private boolean phoneConfirmation;
+
+    public ImmediatePayment(Account account, Order order, float total) {
+        super(account, order, total);
```
Comment (GalAgas) [inline src/ImmediatePayment.java:5]: should get and initialize phoneConfirmation?

File: src/Account.java [hunk 2]
```diff
@@ -12,18 +12,38 @@
 
     // Associations
     private HashMap<String, Payment> payments;
-    private HashMap<String, Order> orders;
+    private ArrayList<Order> orders;
```
Comment (GalAgas) [inline src/Account.java:15]: why the change?

File: src/DelayedPayment.java [hunk 3]
```diff
@@ -3,4 +3,13 @@
 public class DelayedPayment extends Payment {
     private Date paymentDate;
```
Comment (GalAgas) [inline src/DelayedPayment.java:4]: should initialize?

File: src/ImmediatePayment.java (hunk 1 above)
Comment (tomerkel) [reply src/ImmediatePayment.java:5]: we dont use that in the switch case, so i guess it isn't necessary.

File: src/Account.java (hunk 2 above)
Comment (tomerkel) [reply src/Account.java:15]: because we dont need to return a specific order, only the last one and it will be easier to return the last index in the arraylist

**Review by GalAgas — APPROVED** [review]

File: src/ImmediatePayment.java (hunk 1 above)
Comment (JonatanMilver) [reply src/ImmediatePayment.java:5]: we can initialize it for our use.

File: src/DelayedPayment.java (hunk 3 above)
Comment (JonatanMilver) [reply src/DelayedPayment.java:4]: same as phoneConfirmation
